### PR TITLE
Code Cleaning

### PR DESCRIPTION
**Pull request by AurevoirXavier**

### CODE REVIEW

**Comment by AsceticBear** [comment]
CI RED

File: node/runtime/pangolin/Cargo.toml [hunk 1]
```diff
@@ -144,7 +144,7 @@ std = [
 	"scale-info/std",
 	# darwinia-network
 	"bsc-primitives/std",
-	"common-runtime/std",
+	"drml-common-runtime/std",
```
Comment (AsceticBear) [inline node/runtime/pangolin/Cargo.toml:147]: Little format issue.

File: node/runtime/pangoro/Cargo.toml [hunk 2]
```diff
@@ -121,7 +121,7 @@ std = [
 	"serde",
 	# darwinia-network
 	"bsc-primitives/std",
-	"common-runtime/std",
+	"drml-common-runtime/std",
```
Comment (AsceticBear) [inline node/runtime/pangoro/Cargo.toml:124]: Format issue

File: node/runtime/template/Cargo.toml [hunk 3]
```diff
@@ -73,7 +73,7 @@ std = [
 	"scale-info/std",
 	"serde/std",
 	# darwinia-network
-	"common-runtime/std",
+	"drml-common-runtime/std",
```
Comment (AsceticBear) [inline node/runtime/template/Cargo.toml:76]: And this one.

File: node/primitives/bridge/Cargo.toml [hunk 4]
```diff
@@ -24,6 +24,9 @@ sp-api                 = { default-features = false, git = "https://github.com/d
 sp-core                = { default-features = false, git = "https://github.com/darwinia-network/substrate", branch = "darwinia-v0.12.1" }
 sp-runtime             = { default-features = false, git = "https://github.com/darwinia-network/substrate", branch = "darwinia-v0.12.1" }
 sp-std                 = { default-features = false, git = "https://github.com/darwinia-network/substrate", branch = "darwinia-v0.12.1" }
+# TODO: re-organize the runtime dependencies
```
Comment (AsceticBear) [inline node/primitives/bridge/Cargo.toml:27]: Why not do it this pr?

Comment (AurevoirXavier) [reply node/primitives/bridge/Cargo.toml:27]: I haven't come out with a good idea yet.

File: node/runtime/pangolin/Cargo.toml (hunk 1 above)
Comment (AurevoirXavier) [reply node/runtime/pangolin/Cargo.toml:147]: What issue?

File: node/primitives/bridge/Cargo.toml (hunk 4 above)
Comment (AsceticBear) [reply node/primitives/bridge/Cargo.toml:27]: Just put it under `#darwinia-network`?  Too many separations will easily lead to mess.

File: node/runtime/pangolin/Cargo.toml (hunk 1 above)
Comment (AsceticBear) [reply node/runtime/pangolin/Cargo.toml:147]: By convention, this dependency should be moved down to `drml-common-primitives/std"` position.

File: node/primitives/bridge/Cargo.toml (hunk 4 above)
Comment (AurevoirXavier) [reply node/primitives/bridge/Cargo.toml:27]: Nope.
I mean runtimes depend on primitives and primitives should not depend on runtimes.

Comment (AurevoirXavier) [reply node/primitives/bridge/Cargo.toml:27]: > Just put it under `#darwinia-network`? Too many separations will easily lead to mess.

Also, in the future. There might be acala-runtime, bitforst-runtime.

Comment (AsceticBear) [reply node/primitives/bridge/Cargo.toml:27]: Another View: The node primitives rely on runtime primitives. That seems more reasonable.

Comment (AsceticBear) [reply node/primitives/bridge/Cargo.toml:27]: > Also, in the future. There might be acala-runtime, bitforst-runtime.

Add `#acala-network`, `#bitforst-network`, this way is clear enough for me.

Comment (AurevoirXavier) [reply node/primitives/bridge/Cargo.toml:27]: I definitely will consider that. But not in this PR.

**Review by AsceticBear — APPROVED** [review]

**Pull request merged**